docs/documentation/helm.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Helm
+description: Helm
+layout: docs
+permalink: /docs/helm
+---
+
+### Helm
+
+To let Dekorate to generate the Helm manifest files for a project, simply declare the following dependency part of your pom file:
+
+```xml
+<dependency>
+  <groupId>io.dekorate</groupId>
+  <artifactId>helm-annotations</artifactId>
+  <version>{{site.data.project.release.current-version}}</version>
+</dependency>
+```
+
+This dependency will generate the following Helm resources:
+- Chart.yaml
+- values.yaml
+- templates/*.yml the generated resources by Dekorate
+- <chart name>-<chart version>-helm.tar.gz
+
+#### Getting Started
+
+To generate the Helm resources, you first need to configure the Helm chart via properties:
+
+```
+dekorate.helm.name=myChart
+dekorate.helm.description=Description of my Chart
+```
+
+Or annotate one of your Java source files with the [@HelmChart](annotations/helm-annotations/src/main/java/io/dekorate/helm/annotation/HelmChart.java) annotation:
+
+```java
+@HelmChart(name = "myChart", description = "Description of my Chart")
+@SpringBootApplication
+public class Main {
+
+  public static void main(String[] args) {
+    SpringApplication.run(Main.class, args);
+  }
+}
+```
+
+Once, you have configured the Helm chart, you can generate the Helm resources under the folder `target/classes/META-INF/dekorate/helm/<chart name>/` with:
+
+    mvn clean package
+
+Assuming you're using the [Kubernetes]({{site.baseurl}}/docs/kubernetes) Dekorate extension, the Helm resources will include by default the following templates at `target/classes/META-INF/dekorate/helm/<chart name>/templates/`:
```

Comment:
If your project declared the [Kubernetes]({{site.baseurl}}/docs/kubernetes) Dekorate extension, then the helm resources the following templates at `target/classes/META-INF/dekorate/helm/<chart name>/templates/`:"

What will be generated if now the project do not include kubernetes or openshift extension ?

Reply:
I don't think we should include all the possible templates that are generated depending on the extension in use. 
However, I rephrased this sentence again, let me know if now it's clearer.